script/release.py

```diff
@@ -101,6 +107,41 @@ def validate_release_env():
         exit("Please get twine via 'pip install gitchangelog' or 'pip install git+git://github.com/vaab/gitchangelog.git' for Python 3.7")
 
 
+def build() -> None:
+    shutil.rmtree(BUILT_APPIUM_DIR_PATH, ignore_errors=True)
+    status, output = subprocess.getstatusoutput('{} setup.py install'.format(os.getenv('PYTHON_BIN_PATH')))
+    if status != 0:
+        exit(f'Failed to build the package:\n{output}')
+
+
+def get_py_files_in_dir(root_dir: str) -> List[str]:
```

Comment:
You can do the same thing as below. (maybe same?)

```python
import glob
import os
files = glob.glob(f'{root_dir}/**/*.py', recursive=True) + glob.glob(f'{root_dir}/**/*.typed', recursive=True)
return [os.path.basename(x) for x in files]
```

Reply:
ah, recursive is available in 3